Jenkinsfile

```diff
@@ -273,6 +275,40 @@ pipeline {
             }
           }
         }
+        stage('pytest tests') {
+          when {
+            beforeAgent true
+            expression { pytest_test == true }
+          }
+          agent { label 'nixos-mayastor-pytest' }
+          stages {
+            stage('checkout') {
+              steps {
+                checkout([
```

Comment:
Rather than checking out here could we just unstash?
```
cleanWs()
unstash 'source'
```
The CI system seems to be "messed up", with stashes going missing, I wonder if this is somehow causing that?

Reply:
These tests are run on a different agent - and my understanding is this implicitly creates a new workspace anyway. We could try stashing rather than a clean checkout. I can't see how this could mess up the other stages, but that doesn't men its not possible.

Reply:
Oh right, it's a different machine... If we don't ever have other tests on these agent then I guess it's ok with checkout, otherwise we could also checkout on a python init stage. But yeah, we don't need it now then :+1: